packages/account/src/Components/icon-message-content/icon-message-content.tsx

```diff
@@ -16,14 +17,15 @@ const IconMessageContent = ({
     className,
     full_width,
     icon,
+    is_disabled_for_mobile = false,
     message,
     text,
 }: React.PropsWithChildren<TIconMessageContent>) => (
     <Div100vhContainer
         className={classNames('account-management__message-wrapper', {
             'account-management__message-wrapper-full-width': full_width,
         })}
-        is_disabled={isDesktop()}
+        is_disabled={isDesktop() || is_disabled_for_mobile}
```

Comment:
Why do we need `is_disabled_for_mobile` flag when we are already using isDesktop() ?

Reply:
Changed